src/hooks/useCreateClaim.ts

```diff
@@ -15,6 +16,16 @@ export function useCreateClaim() {
       const did = localStorage.getItem('did')
       const ethAddress = localStorage.getItem('ethAddress')
 
+      // quick fix for "Ceramic instance is not authenticated" error
```

Comment:
this is the part about ceramic, right?  can we move rest of stuff into separate PR?  What is rest of stuff here?